com.stansassets.scene-management/Editor/Visualizer/StateStackVisualizer.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine.UIElements;
+
+namespace StansAssets.SceneManagement
+{
+    public static class StateStackVisualizer
+    {
+
+        static readonly List<IStateStackVisualizerController> StackMap = new List<IStateStackVisualizerController>();
+        internal static Action<VisualElement> StackRegistered  = delegate {  };
+
+        public static void Register<T>(ApplicationStateStack<T> stack, string stackName) where T: Enum
```

Comment:
stackName parameter can be optional 
```suggestion
       public static void Register<T>(ApplicationStateStack<T> stack, string stackName = nameof(T)) where T: Enum
```